calicolipidlibrary/MIP2C.py

```diff
@@ -45,40 +45,6 @@ def theoreticalDigest(self):
 			return(FRAGMENTS)
 
 
-		@classmethod
-		def generateLibrary(cls,target=None, mode="pos"):
-			if target: handle = open(target, 'a+')
-			if mode == "pos":  
-				adduct_set = self.pos_adduct_set
-			elif mode =="neg":
-				adduct_set = self.neg_adduct_set
-			for c1 in self.chain1_ranges:
-				for c2 in self.chain2_ranges:
-					for adduct in adduct_set:
-						x  = MIP2C("MIP2C",[c1[0],c2[0]], [c1[1],c2[1]], [c1[2],c2[2]],  adduct=adduct)						
```

Comment:
This line was in the first definition of generateLibrary(). It defined a customized chains variable. The definition appears to have a syntax error: the MIP2C constructor expects `(name, chainsList, adduct)` but receives `(name, chain, chain, chain, adduct)`, maybe a typo, possibly related to in-line square-bracket nesting.